vivo-project/Vitro
# [i18b sprint] 3760 translations loading

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues/3760)**
[VIVO PR](https://github.com/vivo-project/VIVO/pull/3788)
# What does this pull request do?
Replaces translation properties storage with semantic storage.
Configuration model to store semantic translations (associated directory name interface-i18n)
All code that became obsolete related to properties was removed.
# What's new?
TranslationProvider provides translations from triple store.
TranslationConverter converts found properties files into semantic data.
Note: translations in properties overwrites translations found in semantic storage.
VitroResourceBundle addAppPrefix was left for backwards compatibility.
# How to test
* Run unit tests
* Verify that switching between languages is working
* Try online translations (in developer panel settings)
* Try defeat cache of language property files  (in developer panel settings)
* Verify log the retrieval of language strings  (in developer panel settings)
* Try loading semantic data translations by putting them into directories:
interface-i18n/firsttime/
i18n/language_tag/interface-i18n/firsttime/
# Interested parties
@chenejac @brianjlowe
